package.json

```diff
@@ -85,11 +85,12 @@
     "@netlify/plugin-edge-handlers": "^3.0.0",
     "@netlify/plugins-list": "^6.2.0",
     "@netlify/routing-local-proxy": "^0.34.1",
-    "@netlify/zip-it-and-ship-it": "5.3.1",
+    "@netlify/zip-it-and-ship-it": "5.4.0",
```

Comment:
```suggestion
    "@netlify/zip-it-and-ship-it": "^5.4.0",
```

Reply:
I think it might be best to open a separate PR / issue to discuss this. the version was pinned down before (possibly on purpose), and I'd prefer to not mix that change into this PR. also, I don't think it makes a huge difference given that we ship `npm-shrinkwrap.json`